### ISSUE TITLE
HTML content?

### ISSUE THREAD

**Issue by alphapapa** [AshyIsMe/enex2org#1]
This looks really neat!  However, does it convert the HTML content to Org content, or does it lose all the HTML formatting?  Seems like a two-stage process would be good, one where first the HTML is output, then Pandoc is used to convert the HTML to Org.

**Comment by AshyIsMe** [comment]
Yeah you're right, it just strips out plaintext.  All my notes were just plaintext with some simple lists with simple indentation so that's all I needed from it.

You can see the line here that's using xml-strings to strip out tag-free plaintext:
https://github.com/AshyIsMe/enex2org/blob/master/enex2org.sh#L64

Probably the best way to implement this in a truly robust way would be to add the Evernote xml format to pandoc itself.  The Evernote format is quite well documented here: https://dev.evernote.com/doc/articles/enml.php
Adding an ENML reader to pandoc should be relatively straightforward:
https://github.com/jgm/pandoc/tree/e548b8df072c4eecfeb43f94c6517c35d5eba30c/src/Text/Pandoc/Readers

Otherwise a quick and dirty way would be to pull out the `<content>...</content>` sections and replace the Evernote specific tags with some equivalent html tags and then use that as html input to pandoc to convert to org-mode documents.
https://dev.evernote.com/doc/articles/enml.php#added 
It's probably quite easy to rename tags with xml-coreutils so it's potentially only a few lines extra for enex2org.sh... patches welcome :P

**Comment by alphapapa** [comment]
I see, thanks.  I'm not a fan of working with XML, and I don't know much Haskell, so I probably won't be submitting patches, haha.  ;)  Thanks though.
